src/pages/creators/index.tsx

```diff
@@ -0,0 +1,282 @@
+          {loading ? (
+            <>
+              <div className="hover:translate-sale-1.5">
+                <div className="flex flex-grid mb-2">
+                  <div className="bg-gray-800 w-1/3 aspect-square" />
+                  <div className="bg-gray-800 w-1/3 aspect-square" />
+                  <div className="bg-gray-800 w-1/3 aspect-square" />
+                </div>
+                <div className="bg-gray-800 h-6 w-24 block" />
+              </div>
+              <div>
+                <div className="flex flex-grid mb-2">
+                  <div className="bg-gray-800 w-1/3 aspect-square" />
+                  <div className="bg-gray-800 w-1/3 aspect-square" />
+                  <div className="bg-gray-800 w-1/3 aspect-square" />
+                </div>
+                <div className="bg-gray-800 h-6 w-24 block" />
+              </div>
+            </>
+          ) : (
+            creatorsQuery.data?.marketplace.creators.map((creator) => {
+              return (
+                <Link
+                  className="transition-transform hover:scale-[1.02]"
+                  key={creator.storeConfigAddress}
+                  to={`/creators/${creator.creatorAddress}`}
+                >
+                  <div className="flex flex-col">
+                    <div className="flex justify-between items-center mb-2">
+                      <div className="flex items-center">
+                        {/* TODO: Add twitter handle and image. */}
+                        {/* <img
+                          src={}
+                          alt={creator.creatorAddress}
+                          className="object-cover bg-gray-900 rounded-full w-12 h-12 mr-2"
+                        /> */}
+                        <span className="text-sm">
+                          {truncateAddress(creator.creatorAddress as string)}
+                        </span>
+                      </div>
+                      <div className="flex flex-col items-end">
+                        {/* TODO: Add nft counts once the api is updated. */}
+                        {/* <span className="text-gray-300 text-sm">NFTs</span>
+                        <span className="text-sm"></span> */}
+                      </div>
+                    </div>
+                    <div className="hidden xl:flex w-full flex-grid mb-2 gap-4 overflow-hidden ">
+                      {creator.preview.map((nft) => {
+                        return (
+                          <img
+                            className="h-28 object-cover rounded-md grow"
+                            src={nft.image}
+                            key={nft.address}
+                            alt={nft.name}
+                          />
+                        )
+                      })}
+                    </div>
+                    <div className="flex xl:hidden w-full flex-grid mb-2 gap-4 overflow-hidden ">
+                      {creator.preview.slice(0, 2).map((nft) => {
```

Comment:
Api always returns 3. can drop the slice.
```suggestion
                      {creator.preview.map((nft) => {
```

Reply:
@kespinola This is to show only 2 nft preview in small screens (Design Requirement)